actioncable/lib/action_cable/helpers/action_cable_helper.rb

```diff
@@ -20,9 +20,17 @@ module ActionCableHelper
       # Make sure to specify the correct server location in each of your environments
       # config file:
       #
-      #   config.action_cable.url = "ws://example.com:28080"
-      def action_cable_meta_tag
-        tag "meta", name: "action-cable-url", content: Rails.application.config.action_cable.url
+      #   config.action_cable.mount_path = "/cable123"
+      #   <%= action_cable_meta_tag %> would render:
+      #   => <meta name="action-cable-url" content="/cable123" />
+      #
+      #   config.action_cable.mount_path = nil
+      #   <%= action_cable_meta_tag(url: 'ws://example.com:28080') %> would render:
+      #   => <meta name="action-cable-url" content="ws://example.com:28080" />
+      #
+      #
+      def action_cable_meta_tag(url:)
+        tag "meta", name: "action-cable-url", content: Rails.application.config.action_cable.mount_path
```

Comment:
Even apart from the fact you're ignoring your `url` parameter, this isn't what I described.

This should be `content: (Rails.application.config.action_cable.url || Rails.application.config.action_cable.mount_path || raise "No Action Cable URL configured")` (reformat to taste).

And no parameter.